pkg/core/blockchain.go

```diff
@@ -714,7 +714,7 @@ func (bc *Blockchain) storeBlock(block *block.Block) error {
 						}
 						amount = emit.BytesToInt(bs)
 					}
-					// TODO: #498
+					bc.processNEP5Transfer(cache, tx, block, note.ScriptHash, from, to, amount.Int64())
 					_, _, _, _ = op, from, to, amount
```

Comment:
I don't think this line should survive after 1425a063c8d0bbe09813a50b2cfe7082baaf29f5.